src/menu/menu_functions.rs

```diff
@@ -124,13 +124,10 @@ pub fn parse_selection_char(buffer: &str, marker: char) -> ParseResult {
                         action,
                     }
                 }
-                _ => {
-                    index += 1;
-                    continue;
-                }
```

Comment:
Is this functionally really equivalent?

Reply:
I believe so since both statements are within the `while` loop. They would be different if the `continue;` wasn't there in the first place, but it is. Just to check, I compiled nushell with the edited version to see if there were any problems. I didn't run into any, so I reasoned they were the same.

Reply:
You are right! Thanks, yeah after your change it is also a bit more readable.